ccdaservice/ccda_gateway.php

```diff
@@ -89,7 +89,7 @@ function portalccdafetching($pid, $server_url, $parameterArray)
     $parameters = http_build_query($parameterArray); // future use
     try {
         $ch = curl_init();
-        $url = $server_url . "/interface/modules/zend_modules/public/encounterccdadispatch/index?site=" . urlencode($site_id) . "&me=" . urlencode(session_id()) . "&param=1&view=1&combination=" . urlencode($pid) . "&recipient=patient";
+        $url = $server_url . "/interface/modules/laminas_modules/public/encounterccdadispatch/index?site=" . urlencode($site_id) . "&me=" . urlencode(session_id()) . "&param=1&view=1&combination=" . urlencode($pid) . "&recipient=patient";
```

Comment:
I wouldn't try to change directories or anything like that since really just a vanity issue, but could have big impacts.

Reply:
This entire commit does that..............

Reply:
This will mess up @matrix-amiel as well as the work I did with @sjpadgett as our composer package extensions install code into the zend_modules folder. At some point those should be migrated, but all of those extensions will have to be updated.

Reply:
yep, I think we are stuck with the zend name directories. When our grandchildren ask us in the future what "zend" is, we can explain the long history of zend and laminas :)